com.unity.cinemachine/Documentation~/TableOfContents.md

```diff
@@ -76,9 +77,7 @@
   * [Cinemachine Shot Quality Evaluator](CinemachineShotQualityEvaluator.md)
   * [Cinemachine Spline Cart](CinemachineSplineCart.md)
   * [Cinemachine Spline Dolly](CinemachineSplineDolly.md)
-  * [Cinemachine Spline Dolly LookAt Targets](CinemachineSplineDollyLookAtTargets.md)
```

Comment:
Why is this removed?

Reply:
This is accidental. I have local versions of the TOC for base structuring work and I wasn't careful enough on this one. Thanks!